README.md

```diff
@@ -418,11 +446,16 @@ Crates tailored for specific boards.
 
 [FRDM-KW41Z]: https://www.nxp.com/products/processors-and-microcontrollers/arm-based-processors-and-mcus/kinetis-cortex-m-mcus/w-serieswireless-conn.m0-plus-m4/freedom-development-kit-for-kinetis-kw41z-31z-21z-mcus:FRDM-KW41Z
 
+### Sparkfun
+
+- [`samd21_mini`](https://crates.io/crates/samd21_mini) - Board support for the [SAMD21 Mini Breakout](https://www.sparkfun.com/products/13664) ![crates.io](https://img.shields.io/crates/v/samd21_mini.svg)
+
 ### SeeedStudio
 
 - [`seedstudio-gd32v`](https://github.com/riscv-rust/seedstudio-gd32v) - Board support crate for the [GD32 RISC-V Dev Board](https://www.seeedstudio.com/SeeedStudio-GD32-RISC-V-Dev-Board-p-4302.html)
   ![crates.io](https://img.shields.io/crates/v/seedstudio-gd32v.svg)
   - Contains runnable examples for this board
+- [`xiao_m0`](https://crates.io/crates/xiao_m0) - Board support for the [Seeed Studio Seeeduino XIAO](http://wiki.seeedstudio.com/Seeeduino-XIAO/) ![crates.io](https://img.shields.io/crates/v/xiao_m0.svg)
```

Comment:
This crate link seems to be broken?

Reply:
Good catch... [filed this issue](https://github.com/atsamd-rs/atsamd/issues/201) as it seems this board isn't submitted to crates.io. Should I remove it for now or wait for them to potentially publish it?

Reply:
@adamgreig they've submitted the crate, and I double checked and all of the crates.io badges seem to render properly.